lms/djangoapps/staticbook/decorators.py

```diff
@@ -0,0 +1,69 @@
+"""
+Decorators for staticbook
+"""
+from functools import wraps
+from urllib.parse import urlparse
+from django.conf import settings
+from django.contrib.auth import REDIRECT_FIELD_NAME
+from django.shortcuts import resolve_url
+from common.lib.xmodule.xmodule.course_module import COURSE_VISIBILITY_PUBLIC
+from opaque_keys.edx.keys import CourseKey
+from openedx.features.course_experience import COURSE_ENABLE_UNENROLLED_ACCESS_FLAG
+
+
+def user_passes_test(test_func, login_url=None, redirect_field_name=REDIRECT_FIELD_NAME):
+    """
+    Decorator for views that checks that the user passes the given test,
```

Comment:
Please update function doc-string -> Doc-string seems to be mismatched or does not elaborate much about method functionality.